File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/idea/safeDelete/SafeDeleteProcessorDelegate.java
package org.stepik.plugin.refactoring.impl.idea.safeDelete;

import org.stepik.plugin.refactoring.impl.idea.AcceptableClasses;
import org.stepik.plugin.refactoring.safeDelete.AbstractSafeDeleteProcessorDelegate;

/**
 * @author meanmail
 */
public class SafeDeleteProcessorDelegate extends AbstractSafeDeleteProcessorDelegate {
    public SafeDeleteProcessorDelegate() {
        addAcceptableClasses(AcceptableClasses.get());
    }
}

File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/idea/safeDelete/SafeDeleteProcessorDelegate.kt
package org.stepik.plugin.refactoring.impl.idea.safeDelete

import org.stepik.plugin.refactoring.impl.idea.AcceptableClasses
import org.stepik.plugin.refactoring.safeDelete.AbstractSafeDeleteProcessorDelegate


class SafeDeleteProcessorDelegate : AbstractSafeDeleteProcessorDelegate() {
    init {
        addAcceptableClasses(AcceptableClasses.get())
    }
}
